internal/backend/db/dbgorm/values.go

```diff
@@ -0,0 +1,59 @@
+package dbgorm
+
+import (
+	"context"
+
+	"go.autokitteh.dev/autokitteh/internal/backend/db/dbgorm/scheme"
+	"go.autokitteh.dev/autokitteh/internal/kittehs"
+	"go.autokitteh.dev/autokitteh/sdk/sdkerrors"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+)
+
+func (db *gormdb) SetValue(ctx context.Context, envID sdktypes.EnvID, key string, v sdktypes.Value) error {
+	if !envID.IsValid() {
+		return sdkerrors.NewInvalidArgumentError("invalid env_id")
+	}
+
+	j, err := v.MarshalJSON()
+	if err != nil {
+		return err
+	}
+
+	return translateError(db.db.Save(&scheme.Value{EnvID: envID.UUIDValue(), Key: key, Value: j}).Error)
```

Comment:
I think we should find a way to limit the size of it

1. we are not a storage service
2. parsing JSON is expensive for large objects
3. We have to have an expiry otherwise we would collect stuff forever


2 is the most important that I'm concerned about
assume a workflow fetches a huge json from somewhere and try to store it and restore it, it can easily block the process with it

Reply:
we'll do limits in the service layer, not including expiry though.

Reply:
also - changed serialization to proto
no need to do json

Reply:
aaaand added size limit
hard coded for now

Reply:
fetching too much stuff from the database bothers me more than sending stuff to the user